### ISSUE TITLE
Image orientation not respected for RW2 files

### ISSUE THREAD

**Issue by cdauth** [nomacs/nomacs#1174]
I am using Nomacs 3.19.1 on Arch Linux installed through AUR.

RW2 files that have been taken in portrait mode are displayed in landscape mode. Other image viewers are correctly displaying them in portrait mode. Here is an example file (sorry, I had to gzip it because GitHub does not allow RW2 files): 
[P1270615.RW2.gz](https://github.com/user-attachments/files/17282967/P1270615.RW2.gz)

**Comment by novomesk** [comment]
How it should look like?

This is development build on Windows:
![image](https://github.com/user-attachments/assets/736fc9dd-81b4-4e62-b7bb-46a5f891b759)

**Comment by cdauth** [comment]
That's what it should look like, this is how it looks for me:
![image](https://github.com/user-attachments/assets/1784a3a1-d00b-4431-8a18-fcd226060a63)

Also in the preview it is not rotated.

**Comment by leejuyuu** [comment]
This is on 3.19.0 (eca2377d), Arch Linux, Qt 6.7.2.

![image](https://github.com/user-attachments/assets/25f21503-3d8c-4765-8a2d-2d20d53d1ef9)

**Comment by cdauth** [comment]
This is the version that I’m running:
![image](https://github.com/user-attachments/assets/ef4a6dbc-c74b-4d2e-a311-ebccc0ca80f6)

Could it be about the settings?
![image](https://github.com/user-attachments/assets/d6948136-83d4-4798-af63-3fded68479ab)

I also tested it with JPEG files, there it seems to work correctly.

**Comment by leejuyuu** [comment]
updated to the same version and config as yours, but mine still looks correct.

**Comment by cdauth** [comment]
I don't know how to proceed. Does someone know more details about how this feature is implemented and how it can be that it behaves differently on different systems?

**Comment by novomesk** [comment]
Maybe the result is different depending on whether kimg_raw plug-in is installed or not.

**Comment by cdauth** [comment]
If I go to the nomacs plugin manager, I do not see any plugins. Or do I need to check somewhere else?

**Comment by novomesk** [comment]
Somewhere on disk.

For example:

/usr/lib/qt6/plugins/imageformats/kimg_raw.so

**Comment by cdauth** [comment]
Yes, I do have that installed.

**Comment by novomesk** [comment]
Make a backup, remove kimg_raw.so, close and reopen nomacs, test again, restore kimg_raw.so from backup.

**Comment by leejuyuu** [comment]
It is. I installed the [kimageformats](https://archlinux.org/packages/?q=kimageformats) package, then the image opens rotated in nomacs.

**Comment by cdauth** [comment]
For me removing both `/usr/lib/qt/plugins/imageformats/kimg_raw.so` and `/usr/lib/qt6/plugins/imageformats/kimg_raw.so` and then restarting nomacs did not make any difference. Although I don't know if Qt caches its plugins somewhere.

**Comment by leejuyuu** [comment]
Try `pacman -R kimageformats`.

**Comment by cdauth** [comment]
Indeed, removing the `kimageformats` package seems to solve the problem.

**Comment by leejuyuu** [comment]
My thumbnail is also in the wrong orientation. It might be related to the Exif orientation metadata.

![image](https://github.com/user-attachments/assets/2af515f6-add6-412d-a46d-c9eadd937889)
